01-filedir.md

```diff
@@ -73,29 +91,26 @@ $ pwd
 /Users/nelle
 ~~~
 
-> ## Home directory {.callout}
+> ## Home Directory Variation {.callout}
 >
 > The home directory path will look different on different operating systems.
 > On Linux it will look like `/home/nelle`,
```

Comment:
--> On Linux it *may* look like `/home/nelle` 

The linux systems I am on don't usually have `/home/username`